src/filters/Sparkline.jsx

```diff
@@ -37,39 +27,143 @@ class Sparkline extends React.Component {
       }
     };
   }
+  shouldComponentUpdate(){
+    return false;
+  }
+  getLastIdInDistributionArray() {
+    const last = _.last(this.props.distribution);
```

Comment:
if you do `const [...other, last] = this.props.distribution;` you wouldn't have to include lodash